prow/config/branch_protection_test.go

```diff
@@ -18,11 +18,168 @@ package config
 
 import (
 	"reflect"
+	"sort"
 	"testing"
+)
 
-	"k8s.io/apimachinery/pkg/util/sets"
+var (
+	y   = true
+	n   = false
+	yes = &y
+	no  = &n
 )
 
+func fixup(policy *Policy) {
```

Comment:
perhaps `func normalize(policy *Policy)` since it seems "fixing" == fixing comparison ? /shrug

Reply:
Done